### DISCUSSION TITLE
AlphaZero training

### DISCUSSION THREAD

**Discussion by madzai** [deepmind/open_spiel#1030] [Q&A] [answered]
Hello,

I was running AlphaZero training in Linux:
./build/examples/alpha_zero_torch_example --game=tic_tac_toe --path=/home/user/Documents/Work/az_example/

The output is below:
Logging directory: <az_example_path>
Creating model: <az_example_path>/vpnet.pb
Playing game: tic_tac_toe
Loading model from step 0
[W TensorCompare.cpp:328] Warning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (function operator())

It got stuck since then. In the folder, only checkpoint-0 can be found.

Any help would be great. Thanks.

**Comment by lanctot** [comment, marked as answer]
Hmm, how long did you wait? It can take a long time for the next checkpoint to arrive -- are you sure it wasn't just running properly without printing output (yet)?

**Comment by madzai** [comment]
Thanks! It works

**Comment by nsvemuri** [comment]
Any visualization tools or what folders to check for signs of progress during training?

**Reply by lanctot** [reply]
Yes please see the main AlphaZero README: https://github.com/google-deepmind/open_spiel/tree/master/open_spiel/algorithms/alpha_zero

**Reply by nsvemuri** [reply]
I appreciate your commitment, lanctot. I got the basic example working!
